yet-another-docker-plugin/src/main/resources/com/github/kostyasha/yad/launcher/DockerComputerJNLPLauncher/init.sh

```diff
@@ -44,9 +44,9 @@ cd "$JENKINS_HOME"
 
 # download slave jar
 # TODO some caching mechanism with checksums
-if [[ -x $(command -v wget) ]]; then
+if [[ -x "$(command -v wget)" ]]; then
     wget "${JENKINS_URL}/jnlpJars/slave.jar" -O "slave.jar"
-elif [[ -x $(command -v curl) ]]; then
+elif [[ -x "$(command -v curl)" ]]; then
```

Comment:
I'm OK with either.  The ones to avoid are either of the unquoted forms.  e.g. under BusyBox sh:

```
/ # [ -x "" ] && echo OK
/ # [[ -x "" ]] && echo OK
/ # [ -x ] && echo OK
OK
/ # [[ -x ]] && echo OK
OK
```

Clearly those last two are problematic.